Makefile

```diff
@@ -687,6 +689,24 @@ pkg/pb/%.pb.go: api/%.proto third_party/ build/toolchain/bin/protoc$(EXE_EXTENSI
 		--go_out=plugins=grpc:$(REPOSITORY_ROOT)/build/prototmp
 	mv $(REPOSITORY_ROOT)/build/prototmp/open-match.dev/open-match/$@ $@
 
+csharp/OpenMatch/Annotations.cs:
+	$(PROTOC) third_party/protoc-gen-swagger/options/annotations.proto \
+			-I $(REPOSITORY_ROOT) -I $(PROTOC_INCLUDES) \
+			--plugin=protoc-gen-grpc=grpc_csharp_plugin \
+			--csharp_out=$(REPOSITORY_ROOT)/csharp/OpenMatch
+
+csharp/OpenMatch/Openapiv2.cs:
+	$(PROTOC) third_party/protoc-gen-swagger/options/openapiv2.proto \
+			-I $(REPOSITORY_ROOT) -I $(PROTOC_INCLUDES) \
+			--plugin=protoc-gen-grpc=grpc_csharp_plugin \
+			--csharp_out=$(REPOSITORY_ROOT)/csharp/OpenMatch/
+
+csharp/OpenMatch/%.cs: third_party/ build/toolchain/bin/protoc$(EXE_EXTENSION)
+	$(PROTOC) api/$(shell echo $(*F)| tr A-Z a-z).proto \
```

Comment:
Don't do this. Have the proto as the input parameter. See
`internal/ipb/%.pb.go: internal/api/%.proto` just below.

Reply:
I've tried but that will throw an error for csharp. The protos are all lowercase and .cs files are in CamelCase

Reply:
Can you put the error you saw here?

Reply:
Oh actually it is not an error, if I changed it to the target below, I will get an error like this because the `%.proto` regex maps to `Backend.proto` but we only have `backend.proto` instead.
```
yufanfei@yufanfei:~/go/src/open-match.dev/open-match$ make all-protos 
make: *** No rule to make target 'csharp/OpenMatch/Backend.cs', needed by 'all-protos'.  Stop.
```

```bash
csharp/OpenMatch/%.cs: api/%.proto third_party/ build/toolchain/bin/protoc$(EXE_EXTENSION)
	$(PROTOC) $< \
		-I $(REPOSITORY_ROOT) -I $(PROTOC_INCLUDES) \
		--plugin=protoc-gen-grpc=grpc_csharp_plugin \
		--csharp_out=$(REPOSITORY_ROOT)/csharp/OpenMatch
```